### DISCUSSION TITLE
I honestly don't know anything....

### DISCUSSION THREAD

**Discussion by Funtafuk** [community/community#55048] [New to GitHub]
What is this all about? I'm pretty much computer illiterate I admit also undiagnosed ADHD which as you probably know is a perfect recipe for getting lost in the vastness of the world wide web. I don't know what to do or how to do whatever or why. Is there even anything of use to me? I remember coming here a long time ago (15 yrs ago maybe 20) looking for some porn files I think...🤔 does that sound familiar?🤷 Most recently I was just directed here by tutanota email said I needed to update and then came here and don't understand what to do so I can check my tutanota e mail.

**Comment by ncerutti** [comment]
Hi there!
GitHub is mostly a platform used by software developers to work on projects together. Maybe you were here looking for some specific software or file back then. For your Tutanota email, I am quite sure that the solution to your issue is probably not here. I would go on the Tutanota website to look for answers. Hope it helps!
